pawan/html/html_assignment12.txt

```diff
@@ -0,0 +1,12 @@
+Q 12.1 The purpose of the ‘action’ attribute of an HTML form is defined as: 
+‘The action attribute defines the location (an URL) where the form's collected data should be sent.’ 
+Would it be reasonable to use the URL of a simple HTML page (eg: http://mywebsite.com/index.html) as the value of the ‘action’ attribute of a form? Please support your answer with good reasons. This question probably goes beyond what you have learned in this section. You may have to do a little bit of independent study and reasoning to answer this question.
+	answer:-if we have to send perticular data to a perticuler page of specific domain it will be helpful 
+		The action attribute defines the location (an URL) where the form's collected data should be sent.
```

Comment:
this is not the answer I expect, "action" attribute does more than this. First understand the form tag, the understand its attribute, and then write the answer

Reply:
sir i cant find the answer

Reply:
san you help me

Reply:
help me to find answer